### PR TITLE
Feature/ras 310

### PR DESCRIPTION
**Pull request by ar4s**
With range and new calendar.

### CODE REVIEW

File: src/ralph_scrooge/media/scrooge/app/common/controller.js [hunk 1]
```diff
@@ -2,14 +2,37 @@
...
+		stats.init();
+		$scope.menuService = menuService;
+		$scope.stats = stats;
+		$scope.getDictLength = function (dict) {
+			if (typeof(dict) == 'object') {
+				return Object.keys(dict).length;
+			}
+		};
+	}])
+	.controller('menuStatsAdapterCtrl', ['$scope', 'stats', function($scope, stats){
```
Comment (kula1922) [inline src/ralph_scrooge/media/scrooge/app/common/controller.js]: this adapter is specific for calendar and datepicker so move it to app.menu.controller.js

File: src/ralph_scrooge/media/scrooge/app/common/directive.js [hunk 2]
```diff
@@ -2,10 +2,200 @@
...
\ No newline at end of file
+scrooge
+	.directive('tabsdirective', function() {
+	    return {
+	        restrict: 'EACM',
+	        templateUrl: '/static/scrooge/partials/tabs.html',
+	        replace: true,
+	    };
+	})
+	.directive('scCalendar', ['STATIC_URL', function(STATIC_URL) {
```
Comment (kula1922) [inline src/ralph_scrooge/media/scrooge/app/common/directive.js]: dito

File: src/ralph_scrooge/media/scrooge/app/common/controller.js (hunk 1 above)
Comment (ar4s) [inline src/ralph_scrooge/media/scrooge/app/common/controller.js]: OK

**Comment by kula1922** [comment]
squash

File: src/ralph_scrooge/media/scrooge/app/common/directive.js (hunk 2 above)
Comment (ar4s) [inline src/ralph_scrooge/media/scrooge/app/common/directive.js]: I don't think so. ``scCalendar`` is a common component - it's separeted from other logic like menu and return only some date value.

**Pull request merged**